server/src/com/cloud/storage/snapshot/SnapshotManagerImpl.java

```diff
@@ -144,6 +145,8 @@
     @Inject
     SnapshotDataStoreDao _snapshotStoreDao;
     @Inject
+    private DataStoreManager _dataStoreMgr;
```

Comment:
there is already a dataStoreMgr. why are two needed? (also underscore names are against naming convention)